<ui version="4.0" >
 <class>DMXWizardClass</class>
 <widget class="QDialog" name="DMXWizardClass" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>416</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>DMXWizard</string>
  </property>
  <layout class="QVBoxLayout" >
   <property name="margin" >
    <number>9</number>
   </property>
   <property name="spacing" >
    <number>6</number>
   </property>
   <item>
    <widget class="QFrame" name="frame" >
     <property name="frameShape" >
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow" >
      <enum>QFrame::Plain</enum>
     </property>
     <layout class="QHBoxLayout" >
      <property name="margin" >
       <number>9</number>
      </property>
      <property name="spacing" >
       <number>6</number>
      </property>
      <item>
       <widget class="QLabel" name="imageLabel" >
        <property name="minimumSize" >
         <size>
          <width>64</width>
          <height>64</height>
         </size>
        </property>
        <property name="maximumSize" >
         <size>
          <width>64</width>
          <height>64</height>
         </size>
        </property>
        <property name="text" >
         <string/>
        </property>
        <property name="pixmap" >
         <pixmap resource="dmxwizard.qrc" >:/DMXWizard/Resources/dmx.png</pixmap>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="infoLabel" >
        <property name="alignment" >
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QStackedWidget" name="stackedWidget" >
     <property name="currentIndex" >
      <number>1</number>
     </property>
     <widget class="QWidget" name="settingsPage" >
      <layout class="QGridLayout" >
       <property name="margin" >
        <number>9</number>
       </property>
       <property name="spacing" >
        <number>6</number>
       </property>
       <item row="3" column="0" >
        <widget class="QLabel" name="label_2" >
         <property name="text" >
          <string>Output path</string>
         </property>
         <property name="buddy" >
          <cstring>outputPathEdit</cstring>
         </property>
        </widget>
       </item>
       <item row="4" column="1" >
        <widget class="QLineEdit" name="toolsPathEdit" />
       </item>
       <item row="3" column="1" >
        <widget class="QLineEdit" name="outputPathEdit" />
       </item>
       <item row="3" column="2" >
        <widget class="QPushButton" name="outputBrowseButton" >
         <property name="text" >
          <string>Browse...</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1" >
        <widget class="QLineEdit" name="dvdPathEdit" />
       </item>
       <item row="1" column="1" >
        <spacer>
         <property name="orientation" >
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" >
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="4" column="2" >
        <widget class="QPushButton" name="toolsBrowseButton" >
         <property name="text" >
          <string>Browse...</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0" >
        <widget class="QLabel" name="label" >
         <property name="text" >
          <string>DVD path</string>
         </property>
         <property name="buddy" >
          <cstring>dvdPathEdit</cstring>
         </property>
        </widget>
       </item>
       <item row="5" column="1" >
        <layout class="QHBoxLayout" >
         <property name="margin" >
          <number>0</number>
         </property>
         <property name="spacing" >
          <number>6</number>
         </property>
         <item>
          <widget class="QLabel" name="errorImageLabel" >
           <property name="maximumSize" >
            <size>
             <width>24</width>
             <height>24</height>
            </size>
           </property>
           <property name="pixmap" >
            <pixmap resource="dmxwizard.qrc" >:/DMXWizard/Resources/error.png</pixmap>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="errorInfoLabel" >
           <property name="sizePolicy" >
            <sizepolicy>
             <hsizetype>5</hsizetype>
             <vsizetype>0</vsizetype>
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize" >
            <size>
             <width>0</width>
             <height>24</height>
            </size>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="6" column="1" >
        <layout class="QHBoxLayout" >
         <property name="margin" >
          <number>0</number>
         </property>
         <property name="spacing" >
          <number>6</number>
         </property>
         <item>
          <widget class="QLabel" name="warningImageLabel" >
           <property name="maximumSize" >
            <size>
             <width>24</width>
             <height>24</height>
            </size>
           </property>
           <property name="pixmap" >
            <pixmap resource="dmxwizard.qrc" >:/DMXWizard/Resources/warning.png</pixmap>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="warningInfoLabel" >
           <property name="sizePolicy" >
            <sizepolicy>
             <hsizetype>5</hsizetype>
             <vsizetype>0</vsizetype>
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize" >
            <size>
             <width>0</width>
             <height>24</height>
            </size>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="2" column="2" >
        <widget class="QPushButton" name="dvdBrowseButton" >
         <property name="text" >
          <string>Browse...</string>
         </property>
        </widget>
       </item>
       <item row="7" column="1" >
        <spacer>
         <property name="orientation" >
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" >
          <size>
           <width>20</width>
           <height>31</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="4" column="0" >
        <widget class="QLabel" name="label_3" >
         <property name="text" >
          <string>Tools path</string>
         </property>
         <property name="buddy" >
          <cstring>toolsPathEdit</cstring>
         </property>
        </widget>
       </item>
       <item row="0" column="1" >
        <widget class="QLabel" name="label_5" >
         <property name="text" >
          <string>Please specify the required paths and click "Next" button</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="selectionPage" >
      <layout class="QVBoxLayout" >
       <property name="margin" >
        <number>9</number>
       </property>
       <property name="spacing" >
        <number>6</number>
       </property>
       <item>
        <widget class="QLabel" name="label_4" >
         <property name="text" >
          <string>Please select which tracks and subtracks you want to extract and press "Next" button</string>
         </property>
         <property name="buddy" >
          <cstring>selectionTree</cstring>
         </property>
        </widget>
       </item>
       <item>
        <widget class="DMXSelectionTree" name="selectionTree" />
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="extractionPage" >
      <layout class="QVBoxLayout" >
       <property name="margin" >
        <number>9</number>
       </property>
       <property name="spacing" >
        <number>6</number>
       </property>
       <item>
        <widget class="QLabel" name="progressInfoLabel" >
         <property name="text" >
          <string>Please wait...
You can stop the process at any moment by clicking the "Stop" button.</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer>
         <property name="orientation" >
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" >
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="stepInfoLabel" >
         <property name="text" >
          <string>Step</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QProgressBar" name="progressBar" >
         <property name="value" >
          <number>0</number>
         </property>
         <property name="orientation" >
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" >
         <property name="margin" >
          <number>0</number>
         </property>
         <property name="spacing" >
          <number>6</number>
         </property>
         <item>
          <spacer>
           <property name="orientation" >
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" >
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="stopButton" >
           <property name="enabled" >
            <bool>false</bool>
           </property>
           <property name="text" >
            <string>S&amp;top</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <spacer>
         <property name="orientation" >
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" >
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item>
      <widget class="QPushButton" name="logButton" >
       <property name="text" >
        <string>&amp;Show Log</string>
       </property>
       <property name="icon" >
        <iconset resource="dmxwizard.qrc" >:/DMXWizard/Resources/log.png</iconset>
       </property>
       <property name="checkable" >
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType" >
        <enum>QSizePolicy::Expanding</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="backButton" >
       <property name="enabled" >
        <bool>false</bool>
       </property>
       <property name="text" >
        <string>&amp;Back</string>
       </property>
       <property name="icon" >
        <iconset resource="dmxwizard.qrc" >:/DMXWizard/Resources/back.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="nextButton" >
       <property name="enabled" >
        <bool>false</bool>
       </property>
       <property name="text" >
        <string>&amp;Next</string>
       </property>
       <property name="icon" >
        <iconset resource="dmxwizard.qrc" >:/DMXWizard/Resources/next.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType" >
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="quitButton" >
       <property name="text" >
        <string>&amp;Quit</string>
       </property>
       <property name="icon" >
        <iconset resource="dmxwizard.qrc" >:/DMXWizard/Resources/quit.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <customwidgets>
  <customwidget>
   <class>DMXSelectionTree</class>
   <extends>QTreeWidget</extends>
   <header>dmxselectiontree.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>dvdPathEdit</tabstop>
  <tabstop>dvdBrowseButton</tabstop>
  <tabstop>outputPathEdit</tabstop>
  <tabstop>outputBrowseButton</tabstop>
  <tabstop>toolsPathEdit</tabstop>
  <tabstop>toolsBrowseButton</tabstop>
  <tabstop>logButton</tabstop>
  <tabstop>backButton</tabstop>
  <tabstop>nextButton</tabstop>
  <tabstop>quitButton</tabstop>
  <tabstop>selectionTree</tabstop>
  <tabstop>stopButton</tabstop>
 </tabstops>
 <resources>
  <include location="dmxwizard.qrc" />
 </resources>
 <connections>
  <connection>
   <sender>quitButton</sender>
   <signal>clicked()</signal>
   <receiver>DMXWizardClass</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>489</x>
     <y>380</y>
    </hint>
    <hint type="destinationlabel" >
     <x>479</x>
     <y>260</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
